### ISSUE TITLE
No longer compatible with ES5

### ISSUE THREAD

**Issue by sulliwane** [exupero/saveSvgAsPng#171]
Hello,

I'm using this package in my [create-react-app](https://github.com/facebook/create-react-app), and when I run `npm run build` I get this error:
```
Failed to minify the code from this file: 
 	./node_modules/save-svg-as-png/saveSvgAsPng.js:3 
Read more here: http://bit.ly/2tRViJ9
```
Could you confirm your package is not ES5 compatible?
Thank you!

**Comment by exupero** [comment]
Thanks for the bug report. I don't see any reason this library wouldn't be compatible with ES5. That bit.ly link doesn't suggest a problem with the actual source code, nor with ES5 compatibility. The Try Babel site is able to transpile the current version of the script without any problems (see [this link](https://babeljs.io/repl/#?babili=false&browsers=&build=&builtIns=false&code_lz=BQMwrgdgxgLglgewsAlAAgN4Cg1qkgZxjQTBgBI0BeNGATwAcBTBENJgDwYQCcYC0AQhoBySABMmIOBCbiRaAGSL2XXvzQAfTbUYs2k6bKGiJUmXIXLMAXy06YACzgEA3DjRw2wes1ZpDCxNTCEDZeXQwpmBUagA-EjJyFHcPfAgiNA4AWwAbDOo0EUcYGAYALgB6SoB3OoA6GoBmet4Ac0qAJgAGXsqc_IJKkXdcdMzxBFg9QpEAHgB-AbQANyYeAkQIKgAiAEZ67p20IgBDUNPcpCZdiAQdhbi5wQARAHkAYQAVAE0ABQAoicVm00H8AKoAIQAMgBJD5oHYAWmqAHUmh9qi8vi80ABlABqAHE0Ac9tUAQA5Y47EplKq1BrNVo8DpEninBjOKBDQlEypkyrYl6VAggvYHcQwcTHADazypX1hvzQEAARgQGIjFABiPYANm6rh2cQAunERmlCMQwDxcgAlJhtTiFSq23IAHWAsp2IlNC2A9QA1AsUD6_QsPShKqM8Na0CAkDAAGK8bKnDQ0bC4XA1VggTrlIqJiAwWr5zoiAA0Hlz-aLIhLZbzIBA1drJBgIAbnIYuTgUAzW36SKbSIQzFLenbOdoXZ7DD7A6HSBHY9KbZrs5YMAXS8H8FXK1C9WyBFHSYQaoAVkxYDOcwQQKW9_2D8Ox0_Sw_cGK2g24HTZ1RRBIMBhEDwbFSMZ4xcAFciYbImFLQor2veISBvTwMhgc4oH0NAAAkvgAWWheDEOQ4htEw9CZDOaACL5CikNLWNxmIHgmAARzAOAuJeBBskpBBJEKJhcgw7Mcy8NBgEEOCENYmBgAklB0CcHgEBqVUmB0gEeC0nhgAAA3OIjSPIpSqJIHh8WJFibJcNAuN4_i5FcNA2gQYhyAwCSbBMlJIPY2CCABDgYHWCBLkKd0MPi6x3XqXJTiIWFQk4N4QGAYpSgYatunQKgSrQbolBUZLUvSzKOGy4AahkSYahSqYVwgepHAQIhitKpE9mguMcK8pgUyTEjAKYL49GTLTsnBO1CibBbJKoBJpJg4bEx4dNMzQN4bzvGB6gAayYOgCFAJNUx2jMCBQDtcHqaRcii4zOCijItgw5a7XqJqspykz6j8j7kM2JBAvQBIisetBT05VTIvB761oTa60zu2Uwa-pBTWC2dZKu27-HQLiYFtCB0ZJghZW6U1Y02ggEAQ-p1iM0zwQgE67hqKmm2p3bqbQPzftyGx6jQVNSwEdM6FVHy0DVJg0DzHgTpkUF8EMo7cjoeogsZlzRspopezfdrKimKKYCRIguNObJLVwKCsCtYbTkM046EhMBW3WL4EEhNKmH1AAWQo1T9kB1ikjsEOINUZE9-XRGdnMOKVugooEGhZB08EZBgAAOABBL26GAKP_Z4Amc22uSE88QojWbuYs5z-o1WzphoWQtonE8uAgyDdAk5inh5aDGg8RgHhNeeuaPkcT2PlE6Ju5z2U4Hxo3ydNxrQm0ruYAQU4q-Tye65sN2meIZ0YBeSbcapmhVNyKs8CuWRP_EQDivWh2TOKxqBw3fvUFYcA9KQgQBwCq7BcgQKgTUGBHAu4hwJLFawEkkHQNgegggTBMG5FlH_bI-N7BgKgN_Jg9QH6l1KPPKOUVgBkPQMIPOYBciSWsIIah1w6GjQYXPOAzDohsIRjAKAjhgCVAAKTkEqOgawDBPaEIyipfhshBEwGEUwsg4j_7qUobOXAOCH4wIkJrD4_YqKOlgKgUhgFTQmNMaojYTANHAC0bQogdAEJOPIegbQcN3HqNLA1Jqx8H5r2yAwAx4hZ7-OiGpHRfwtLMD4HQYhYBDHZHUnvE2PAqa-AIiAkqIQojyHsKsagpUIBcMkjRFwlJTiUmAGEpgyYrgZmACsYxCwypoCLCsWMrt3aZAfk_ViEMChvwkp_Hxn9GpSkcJ_RwTA4BtBKAAzAHYiY4Nwm0VpSE6miD_CIMmRSqYbRzCspwRZ7mOBqVM5-sz36LJoZ_EQTzLlblMRsrZJQiyAu2dRHQryZlbA-V_a43zQUlEuR2V225ciEM8N4cxo1ISoPQLcu-mAOCfzoMsuAqz1mbLBXYGgWKYA4tgagI2uB97FL2aYu5ZKHlZDQEGVWnK1lw1wAi3caAp5oGFXDFFOYb4u0GpnLiAJoDr0KOIDMpx4gdkkPgSQ4J7TwiEtwWQES4bIW1UwXV-q4nXAiaq3CD12VPS4n2U4-FZFyO9N0JEABOUuSJkymgwJ0Gw0Y2if2ALtaRn8GB7F2fi0xmcoCFFnvPCAbRF5CWXqvdeplugcD8tGqGTLTEsqpomipRQ5EKEGSIORnQACsCgixQCLS7e1OZgoTJtPPQOkIrhqjivPOOs5M6byYMmzWhRD7NXqBmK8wBbRwHqJqfsKlqyXNlHsXeQD4zZEmuO1NA7F3LrgKuqs676ZLqXKu8o67N2XpXblVw57TTbuGtXGOdk856TQOXDkPto7rCrj3fdaaEKpqcHXAlRdf3e0KPnNAhdSxlwrlXADtcjYN2AE3OALch5oHbqOkDKV-6D08CPPFcNoMV23i4mghGRGpvqNIrNkgGHADgJBtAMqcwlt0jpXtc7ZTvvWKaT-GBSlFl3UhEDwaxlyvjLxdYqc5ILJOBJI6vBY37O8IIQhCFYCaeNhTYpRs57yzjbxnBinJ54nUwZ4yemNO1xqTg9xVERJiWwYgtzpYPO0Os3QWz-nT4ObsyFzjdgDzSNUoZCj7Lxgs1oTUT2yATIZRWJcMlaAPh4jxGp4LvBER-Uc_ZmwOwTKf3ZhFkKt8hoTFGkdD4BACDS2IDQHgXCVZo1udUNAJEMzSOy7l9GpZRwupVh1hCAhT6qDejFSS_ZuYEHqB2XrAABMcIBxtspzL138PAoBFiuIOXIuVS7K1yH8tA7pgB0gYAQBkTZlttDOPAKATGhKikqKcC7lQVj6kqOCAAXpSJEUDrz2j2ASBAAANcEnQ3hQCRAAdmhAQNUawCAAC1USdDqgARSRBwAglJuJvDDsj7Ida60vFLo0CsnHevcYJRG557XOtMea18D6kjovukdM6DgnHM7xTfqz-BrON0UJoiIdOuAiaCHijRZKrPZEAD1bWnCqMEnQovSoiB-6QXcapUrc0uUZymRtM7gG4StUB7LkpnD4AQVEJ6ZEiHqPUYY6BBkmT8pN2hjguIgBsJUT3lQ_LukCnDIsjvcLO9d04XKXvze-_9xzoPUhQ8gwwFHkyMfrt2kKcZm5cMoqRSLAHznBBueRX-bObau0iwP1axNJC01mCzSEitUAjSVooHrzmd0RYbe5BWsiuTtXM4yEW54oCTBc4IIwukoSLhaGXFOx2GD-sQBzXAQFoLTnS7cNyoBU4zpLkoARgwdj8-h2mKbpntgNAz_Oh0Xo0RBjKR4lyndhkdQWpmR2gBQvVQD-hFsTpqwign9zdml5939GFP8WFihg9Lki0Fcn8rkS8wQ5o196guJmZcg1hgAGluFON5dvA4JkZilLhbtg9jE40hVg8eU34n9_pap6oRAFhzdy1-o0Bq1uDhkihFBzdeURAYAqAFBeV4MXgMxogr8Mtchcl6pyCuM4ZeN4MV9d1CEYgCCWY1hP4uJbxYAtMHU6tiBBwIAMtF9JgoAwBlImMHYopHIIkRBLDrC0DBVzDPBshQQv0dJYR59GUvDAI00oAtJms3h542gZBZhzgkA6BshSACA5dTFQil0DtCgn8W0fC01rhDJCs35dkjCjoSDv0DIOYTI14uFxAFZiAelai_In8oZVD5dfDWgIAGjChYhusvCxhzhrDGh-VCh0inkcj-irC0pOpKUSgRj2jhVxi8ABipiYkkwPpcpOgIh6hxAOQahAjz9ohQjP5uhjjWiZJ4DCFdFECxFv9f98p_8mQWhgC9hQCvVwCZBID4VUDFlljltT5ZDcJdVoRT959KgGBU1L9Fi9CiDog55ckzipVZxZNkU20O0CVBxpE5BWtF8MApVp9OiLBsSlokxc4EgtC8CN9gAOwntr8roUI1o4ZNDcCdDgBoSDDjZjCYBTCHUiYMSNlxBsTZQmx6h3QKFeM2Tog-SsSSShSkwRS7Qt0vD5UeI4Nv0YcyJCJ8pHRFMiBgizDXIZ1xBxAAQ1hSxoQXBPpAMRAGioCejAEzDcBesvg3h3giwT00A5Y0BLhmYlYVY1YnBPBiAZtkt3TIB4BJInsvSuIExGl5YGi5BfTtomA-i0BesMwoo4l4AD0ZsOQiB1g4AgcVYnBEJJZVI6FJYwB0U7sHtqhJA1grhMlTwEAgc4BuFTgWQOhkIkRwQ8RKhbChhUQmA1RKhS4_hYRKhWtkxxtbMuSUzrckwMpg5CFw5ChPY_1fYa4e0Q5w5WSeJ8CF9DVCEziCVfocMaBhTy9jonVUpXV-cnROBP4TIbsdhNdyhRY5TG8MwoI1Rtyw4qx3zSxFzfyysUAgogwRAPQIBUj4sXV-TBThTRTiTSwFo4AoSF99DohTyESziDTTgjSTSqJzS8zZBjIRB2ZeAoCut7SHSEtWZktilTIpzWyEyZsuiBZd8hIiwALjo89KtjyljMSBSZSEKFS4NGk0LCDiDSDchsKcjcL8LTSYAiLLTSLDc-BKL78zDaKksUtTJzt1AEyGiJ12K5ouKMARLxYKt2B-KJSSDGlZKvCDSJxkJcoiQAQvgoCLKcK9yCDDypoZhRA1zvZhMeBoKeM9zCFQg9TTFg0OxUT6hizkAmwmtSSRsYAUrnpWy3pgA4E0Yhd6hrwEAZBcpITasm4pTxB7ROtF9pLQpho_EEI8QNlRoqqptujuSKC5IKrWqF8sDTZurqri9-rYK5AerF9t901shWEph7CqIl16BGrmrSZaSCAlrNKcwzMdt2VeNVqmBRpq8xqi1IsBtnlVI4sHVtLGhdKTIkkps1r8Aai6jfS0B4zxAzLdr9rmjDZJUUTJ9O1sJZ8UrujVMJxDwMgOrvC40SsQtHR0wGBB9cAki_4QBAsFqmAEaPTRIvA6AUqMantkVUIGAwaBAaJcSrd4xnQSK5CgaaAkbsagaaIYhobeAo0Ml1h4Ber1qCU9NChmaeBYbOR-D8snMBab8-b0Aiw-ai1M4GAMlF86aUbbqVZBkFbUbkkOk2a-AoF7ohDZbnKtaF8i1eM_cMA9MbAMB805abAbBIKTIjZkTh14weRF9ZQGZX16sopYAiSaBSl_BIzy0xBMojBLByatokxlKkKNAaJXajYGqx0lqxrUBnpeAARYLWTqquaMU5JBAA8daS0i0JqOKprc6r9tpU7osA9M7OqfA9B_Aq846TAigzBg6IgtqeTvAAsYV66wteBa8uT0Bnb6h4lVrgBKb1hqbmt07WY-a-7DCOc47q8-6CkUyJJ0UiZJBPaxoZZ4Eq9na-7ecZFKg1cNskwxt8IlFzqHTvCBYaAN7GtmtWsp6mB-Lq6mx0AmxlKh7KyZE37Fi7BV6VZB7h6ZFd6ucPoETJUIs0TwrsCZ9CSSS6SlKLSr9ErEGaaEhB7CriqIKoL0BeVkrmtr5BojdyAh6nVPYx0QRgaP4SAiatgCBf5rgIbXI-IBIhI_N35hd4w40EIQA2syoMbT4tQaATi4YnlHkMbhUQUMaCATsutSQMbfLCA4A1glpvT0aCaaBQb6GakybatmVrkAaLB9jnRLpUlUG7S27cByqaFxJEEfEOG4Tn7pbuGnkKUgUYBqURpH43l6GYUllVZ1loHCZMVEEjkTkVYOEigLlL60jQmdF6UhcrH40aEl0hFriDFco55zgnw0woCfEECRExEsmOQMhG9Ll9znVXVKhsmMhUooovR6gAAqSMaMb5Uqq-zOP8FVGahw8IpgOQlwmAW4vKekaoAAxoJ41kLoXoboECNoNdGJxYv8GdRcZCcQZeVs8QbxGhfinxXmkEHI_-tFFWRgjOQgRLNmAo0i4RRCImlihAY2TKOyO4CAJEPkBBSib8SrGStCkvI5n6uGApy4j_YpkQNYDYLYKAkQMkTw-LVJkFjJ5AyBPBDgKA2UXhmAT-IR0lclcVGYmAU0Aqoq5ABQDpuJ7Ogp-hJF6IEQAYDIS_WFbRRFopr_H_elhhooDl6Fv_cZx4jsmZvoJZyjHTKl9J1l5Ajl8oDgCAxl4F8V_RKKW4jl75KVmVj4nl-4vlwAqZjoF4sA9Vs3YJmSbwJRr6NYWJ4dVJriJItYUFzJn5flOFi661xCBAO1ml3_Al51-FgRFlxV2l2WhfdYO1zUI6e0FcaFjgCaCAH4GNj0vamAH16VBBdFM5-Vq4iV2l35XFgMppk4ORs4jN-15A4VKA4VNAfN2Ry4ZxwF9lQuvfApg_HungY_U7RsXgSlCAA6TktABIJpy_ZOngcun-ztrZbtw6WAKurO-SJM8dnto6QpgN3Kbl9SFMudtoCdzktJzNgN5VvIDIVVg9lI-FLVxkHVgV_Vt4jgEoPIZN1tQaWcXjOB2QFKmFbRwgFBjZZAZ26drptGnpuwvppwpgIZ3KOO-9x8NGndkt2l0paFy8yoZ2yD38aDmQEi4iMiQoEyZ4WUD4WnL4UuWUSCvyZ2m2iAU0c0O2_R-NeMQwGw3pua_pwZ6yVw-jlDgIKQZbXsdZzZ3IbZuOot1J-idmyEKQTt1hLjz5ARaQDYdK5wfj41x24aI3WOW-xj0sRwgZ5w1j1dP-FYDj1TngVZycDZhT7ZnxIt-MI3TUvIVCAxYz9D9YTD6ESpm86ISob_D0cQIMcoJ_SoGIo9wYaViA3YXl89yZy914947mY4Q1k6Pz1ApTk1uSSYWQSIa4YAGzmAPIXNtZfFjxs4gBi3VlckwhfcyS6IDAW93IIsbL3LvlPF4VB2mKqBv63AEhpdEEUuAgAE04FC6hz-D9w9gIJh9alh9yQSYSbNNSUOzIAgrhfhzroNtzPEEEd9uhz9uGBK79mIarnL8WXZEyV81_JgeZsCPIVwH85cv8vyKdY-NUU-c-BVJVSQaa6YZgIMGr9SQtbTVLsb8UhfRbnblytL2tp9wxhb16drxICgLrtoHrv4VNAb-ZGh4bzlsH5hniVhpgKbjh2b924gONU1deHgDveRkQU7sEiEjGknyQHgN4TbuZMqeoYuDGn8qAE6NoLSCQNeK4HgDG9wtoTR2h4m3R8ZZTzIdICFmAQOJHvwuSAgA7ZZIJzOhNX4wD2azT5jnTr51ddwtKSDhNNYyKQoA357UaNeUsdYkQTYo3-MBgOADgCSCNw8SdKJFqOsgcJgP4J3l3lcGpAaGjiYwYp5SdSttAR353h0FcaW346YjxrIiPqP_3w8OPyY5bBesPmgE283oY1ZGwBgDgajmCjP-a5JBPsFbD0j-P5rovkv2j63yKHdr4Up3JnaDpP3mPw8Y43vyPrv13xAPvoqR9lL7xAYtoAeif8fjPz-JXqAVQ4r9IS87Y3Ykx6Ief4f5LomDnrnnn0IPnwzdNk346NoPtS4WJbgTYKKRn8esG2YSQIgZOMG8cCFsKzaFfl6XIJWyOF1Pf0gA_izF4A5Fl-POL_vYhUgnEBGSxMvm4xgGDFhUqhG-HDCbjgk2gRaTamczQFm94-_xNVECVUivcA4egSrEQIZ5M97oR1JYlIlOrP1km1dGun4DYC2Y7C88egBUUKxRNA6VSKwCoBVj0RcIjEfwCwIXTsDrmOudgPUBiinIA6IgtgXQA4GhVLWjfQgrQnIqkVHQzzBMh81O4CBLCdwROCrGaidEz4kgWorEScDOQ1QWkGoIQmM4ccDG_zLwsc3RSaRtI7AI6nDDB4dJU01WP7jPxBDoBlu5DLiGtzaAbdiaiyJADLzl6-CjYxXTrn-B659cUKEQ-hp_AXRq9YId-fwmgHX7RVCY8BJAF0SKIYRpe7NWIeENO6fxTu-fJwDUPgKID0CRQkikZHaplCLmrMdQaZC-AbJoyyWAQOZHUEvVTBE6YsgEDVQIY9UXpQYVTFO4kASkGydGNwm0gTo-QJHDAPdxaizo1Q86eeEujfDRBOg-oH7rf0WEqxEwKww-KCAgK0BHmhCFWNYO0h2CRAAgP-OfjuCbACAGwhdL93ZS1D5-h6e2kQ1qyddjBJQuSNIPRqF4OM61ImDFEgRtBZ0xnM8HiFOBrAoiZwgTGqHQAIitkyI08AQDREYjyByEbEXsLgA9o-0FIgfKqEdjg8cwxXONF03RG9wPimvYDtp1A66dcopwSDkTBEDgjTBCgWIgQFZHmluYyg38OKI-LbFtIJgvCnBnpE5ExRawCUSdHL6sw_4y6WDKIFeZMB3-o3IDnNTVCiR9YPHA_uZ2ACqi2RkonIpgLnLxgTcV4Q9FSLnQLo9m8YUXFMNc468mAC7WAAQOdE4jlmMo7mJ1GYI0B3QoYtUbKKQDUIBwJ0doWjBYYL5dEEAM_GDVmj0iYguyIEvuRWAIAzoAYmAAQPdDL0zCx1GgSpilHnMMglzeiqS16FWCbBdg0bgvieoEAwAi4dQLRCOg-jlsUsDfBOl353CTgDGUEPmIcEnAwxGop_ECOcFeEbR6opjG-BOgFDTEthLXsdFNHiB9YNrd1kwD44CdZxSAk1Cc3oF8oj4LUZysgAXTfIAA-hmQKjfJWIYAH8jwCoB3Aqwp8FmB-K_EIAqwr2SsgBMg7M41CsqUEUkCXSsiwhg3OkUhCG4UCsebkNhtNze4E9ZwiQ7rr13wHzw0hhAcXhkMHRowwR8ohoiQXpHESOMIIjwIkNgk4T5e8EqEUhOJooSceePGbr8w7DYSEeBAeXqkJBoUCiJMIjCGRL5gUSWJMIisVxncDBpgiQAA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&lineWrap=true&presets=es2015%2Creact%2Cstage-2&prettier=false&targets=&version=6.26.0&envVersion=)).

Can you supply a more specific error that indicates what the problem is?

**Comment by Jephuff** [comment]
The problem is likely that create-react-app doesn't run node_modules through babel. 
A work around to this is to override create-react-app to transpile specific modules using [react-app-rewired](https://www.npmjs.com/package/react-app-rewired)
with a `config-overrides.js` file that has something like 
```
const path = require('path');
const fs = require('fs');

const appDirectory = fs.realpathSync(process.cwd());
const resolveApp = relativePath => path.resolve(appDirectory, relativePath);
module.exports = function(config) {
  config.module.rules[1].oneOf[1].include = [
    config.module.rules[1].oneOf[1].include,
    resolveApp('node_modules/save-svg-as-png'),
  ];
  return config;
}
```

**Comment by sulliwane** [comment]
@exupero to reproduce the error:
```bash
npx create-react-app my-app
cd my-app
import saveSVGmodule from 'save-svg-as-png';  # this is a line to add to react app project
npm run build  # this step will fail when trying to minmize
```

I suspect the problem comes from the arrow functions, that are *not* part of ES5 (but part of ECMASCRIPT 2015 / ES6).

As @Jephuff suggested, one workaround could be to either "eject" CRA, or "rewire" it.

Or to offer this module in two flavours, the default one that should be "ES5" compliant (transpiled to ES5). And a ES6+ flavour for people who want to import it...

@exupero would you consider making a transpiled to ES5 version the default one? And adding a *esnext* key to the package.json for those who want to enjoy ES6+ ?

here is some lecture on the topic:
* https://github.com/bookercodes/articles/blob/master/how-to-build-and-publish-es6-npm-modules-today-with-babel.md
* http://2ality.com/2017/06/pkg-esnext.html

thanks!

**Comment by exupero** [comment]
I'm hesitant to include a second, ES5 version of this library as a checked-in file in the repo. Having to build and check in a separate version makes it more difficult for contributors, who now have to distinguish which file should be edited, as well as building and checking in the transpiled file. It also creates potential for the two files to get out of sync. ES6 is widely supported natively, and if something in your build pipeline requires ES5, I tend to think it's your responsibility to include the proper ES6 transpilation step. It sounds like there are common solutions to handling this in your app.

**Comment by MilosRasic** [comment]
We're getting an error in runtime in the same line in IE 11 exclusively, which I assume doesn't support arrow functions.

@exupero Publishing completely transpiled ES5 code as "main" file is a common practice in browser javascript open source. "module" key is used these days (last I checked at least) to direct bundlers to an ES6 build which should be compatible with most modern browsers.

If you're making a decision to not support ES5, which is still strange, at least make it in a major version release. This way you break the internet.

**Comment by exupero** [comment]
Can you supply more information about the error and line of code? I'm not able to test in IE 11, but [this ES6 compatibility chart](https://kangax.github.io/compat-table/es6/) indicates that IE 11 should support ES6 arrow functions.

Thanks for the additional info about supplying two versions of the code. I'd happily look at a pull request to see what's involved.

**Comment by MilosRasic** [comment]
idk man, I see a solid red with 0/13

I'm unable to test as well atm, but our Bugsnag caught simply "syntax error" from many IE 11 users. Here's a sample of device data from one of the errors:

browserName: IE
browserVersion: 11.0.0
locale: en-US
osName: Windows 7
userAgent: Mozilla/5.0 (Windows NT 6.1; WOW64; Trident/7.0; SLCC2; .NET CLR 2.0.50727; .NET CLR 3.5.30729; .NET CLR 3.0.30729; .NET4.0C; .NET4.0E; InfoPath.3; rv:11.0) like Gecko

**Comment by exupero** [comment]
You're right. That compatibility chart is bunk. It's showing different results in different browsers, which I don't think should happen for any column other than "Current Browser".

I'll work on restoring ES5 compatibility.

**Comment by exupero** [comment]
I've created #173 to fix ES5 compatibility. Test it out and let me know if you run into any problems.

**Comment by MilosRasic** [comment]
Just tested with IE 11 from a Windows machine. Confirmed the problem with the latest version. Tested the fix/es5-compatibility branch with npm link and everything works fine.

While it's definitely working with the fix, I'm not sure babel-polyfill should be made a dependency. Most project using babel will already be including it, either through import or bundler config. What happens if a project uses babel 7's @babel/polyfill package? Duplicate polyfills in the bundle probably. Not sure if there would be any conflicts.

There's a pretty good discussion about it at https://github.com/w3ctag/polyfills/issues/6

**Comment by exupero** [comment]
Thanks for the feedback. I removed babel-polyfill.

**Comment by sulliwane** [comment]
the new transpiled to ES5 version works for me as well. Thanks a lot, and looking forward the NPM release :100:

**Comment by exupero** [comment]
Published v1.4.3 to NPM. Please re-open if you encounter any problems.
